WindowsServerDocs/failover-clustering/troubleshooting-using-WER-reports.md

```diff
@@ -395,7 +395,7 @@ To identify why the hang happened, open the dum files. Then run the following qu
 
 ![Output of running log query 2](media/troubleshooting-using-WER-reports/output-of-running-log-query-2.png)
 
-We can cross-examine this with the thread from the **memory.hdmp** file:
+We can cross-examine this with the thread from the **memory.dump** file:
```

Comment:
Please note: "memory.dmp" is a very specific file name with the specific **file name format 8.3** (MS-DOS compatible filename format).
```suggestion
We can cross-examine this with the thread from the **memory.dmp** file:
```
Because the file name is this exact 8+3 characters long and is created by the Operating System as `memory.dmp`, we need to keep the name as such.